docs/Features.md

```diff
@@ -0,0 +1,25 @@
+# Features
+
+## XML Tag Auto Close
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides an `auto close` feature for XML tags:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag 1](images/Features/AutoCloseTag1.gif)
+
+There is also an autocomplete for adding an opening tag which adds the corresponding closing tag.
+
+![Auto Close Tag 2](images/Features/AutoCloseTag2.gif)
+
+## Hyperlink From End Tag to Start Tag
+
+For quick navigation, you can use `Ctrl + Click` on a closing tag to jump to the corresponding opening tag.
```

Comment:
It works too to jump to closing tag from opening tag

Reply:
Is the wording tweak enough or should I add both jumping from opening to closing and closing to opening?

Reply:
I think your demo should see the both usecases and your description too.